java/ql/src/experimental/CWE-532/SensitiveInfoLog.qhelp

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Information written to log files can be of a sensitive nature and give valuable guidance to an attacker or expose sensitive user information. Third-party logging utilities like Log4J and SLF4J are widely used in Java projects. When sensitive information are written to logs without properly set logging levels, it is accessible to potential attackers who gains access to the
+file storage.</p>
+</overview>
+
+<recommendation>
+<p>Do not write secrets into the log files and enforce proper logging level control.</p>
+</recommendation>
+
+<example>
+<p>The following example shows two ways of logging sensitive information. In the 'BAD' case,
+the credentials are simply written to a debug log. In the 'GOOD' case, the credentials are never written to debug logs.</p>
+<sample src="SensitiveInfoLog.java" />
+</example>
+
+<references>
+<li>
+<a href="https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html">OWASP Logging Guide</a>
+</li>
+<li>
+<a href="https://cwe.mitre.org/data/definitions/532.html">CWE 532</a>
+</li>
```

Comment:
The query help builder will interpret and automatically add this reference to the output HTML because the `.ql` file has a `external/cwe-532` tag. So this isn't needed here.

Reply:
Sorry about the typos and thanks @felicitymay for your help with the clarification. I've committed the change to remove this reference.

Reply:
Thanks. The addition of automatic references is a little unexpected, it's mentioned in https://github.com/github/codeql/blob/master/docs/query-help-style-guide.md#referencing-potential-security-weaknesses, but isn't very prominent.